client/src/main/java/jass/client/model/SplashModel.java

```diff
@@ -120,7 +120,7 @@ protected Void call() {
                         }
                     }
                 }
-            });
+            });*/
```

Comment:
```suggestion
            });
```
Uncomment function again (was just for testing)